recipe/meta.yaml

```diff
@@ -1,6 +1,6 @@
 {% set name = "pystan" %}
-{% set version = "2.17.1.0" %}
-{% set sha256sum = "d4f91b5300c002448c4141abb806264f929fffed4fb72b9dbdfd2def83529b78" %}
+{% set version = "2.18.1.0" %}
+{% set sha256sum = "f8d50218d00848bff7cc9f33471eb5c0cf0f84af0a807963349a7931bb782b46" %}
```

Comment:
I'm ok skipping 2.18 to go straight to 2.19

Reply:
yes, I'm just trying to get m2w64-toolchain working first.

We still could build the wheels 2.17 with MSVC, but 2.18+ can't be build with it.

Any ideas how to get libpython/m2w64-toolchain working correctly.

Is there a way to use msys2 channel (or m2w64-toolchain for conda-forge)?

Reply:
Ahhhhh, I see. I am a complete n00b on windows. @mingwandroid is the expert on these things. He may be able to help if he has time. We can also ping @conda-forge/help-python-c to see if they have anything to add.